server/redis.repository.js

```diff
@@ -76,6 +78,20 @@ const dislikeEvent = async (roomId) => {
     return -1;
 }
 
+/**
+ * Deletes room in Redis store by roomId
+ * 
+ * @param {String} roomId - The room ID.
+ * @returns {Integer} 1 - If room is deleted. 
+ */
+const purgeRoom = async (roomId) => {
+    if(!roomId) {
+        return Promise.reject('Room ID was empty or null.')
+    }
+    const numKeysRemoved = await delAsync(roomId);
```

Comment:
will this method always return 1 if there isn't an empty or null roomId inputted?

Reply:
Yes. It will always return 1 unless there is an error within Redis which is highly unlikely for our use case as it is very simple. Let me know if you have any other questions/concerns.

Reply:
Right sounds fine to me. Approved.